src/client/pfs/pfs.proto

```diff
@@ -169,6 +172,7 @@ message ListCommitRequest {
   CommitType commit_type = 2;
   repeated Commit from_commit = 3;
   bool block = 4;
+  bool include_cancelled = 5;
```

Comment:
calling it `cancelled` seems to imply that we only want to list cancelled commits though?